ironfish-cli/src/utils/table.ts

```diff
@@ -132,3 +133,31 @@ export enum Format {
 }
 
 export const TableCols = { timestamp, asset, fixedWidth }
+
+const { 'no-truncate': _, ...tableFlags } = CliUx.ux.table.flags()
+
+export const TableFlags = {
+  ...tableFlags,
+  truncate: Flags.boolean({
+    description: 'truncate output to fit screen',
+    default: false,
+    allowNo: true,
+  }),
+  'no-truncate': Flags.boolean({
```

Comment:
Looks good, the relationships section seems sort of funny but I am guessing that is some necessary magic for oclif.